src/apps/deskflow-gui/dialogs/AboutDialog.cpp

```diff
@@ -89,5 +89,11 @@ AboutDialog::AboutDialog(QWidget *parent) : QDialog(parent)
 
 void AboutDialog::copyVersionText()
 {
-  QGuiApplication::clipboard()->setText(m_versionString);
+  QString infoString = QStringLiteral("Deskflow: %1(%2)\nusing Qt: %3\non: %4")
```

Comment:
Nit: For consistency, I think this would be an improvement.
```suggestion
  QString infoString = QStringLiteral("Deskflow: %1 (%2)\nQt: %3\nSystem: %4")
```